.github/workflows/release.yml

```diff
@@ -5,39 +5,43 @@ on:  # yamllint disable-line rule:truthy
   release:
     types: [published]
 
+permissions:
+  actions: read
```

Comment:
I'm not entirely sure this would be needed but [docs](https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#permissions-for-the-github_token) are unfortunately unclear on what particular permissions each of these give but I believe that `actions` is more targeted for managing actions and/or workflows (so e.g. dispatching a workflow run).

IMHO it's safe to remove this one here.

```suggestion
```